bentoml/deployment/serverless/serverless_utils.py

```diff
@@ -129,10 +129,27 @@ def generate(self):
             ],
             tempdir,
         )
-        shutil.copy(os.path.join(self.archive_path, "requirements.txt"), tempdir)
+        requirement_txt_path = os.path.join(self.archive_path, 'requirements.txt')
+        shutil.copy(requirement_txt_path, tempdir)
         model_serivce_archive_path = os.path.join(tempdir, self.bento_name)
```

Comment:
not related to this PR, suggest update naming - s/model_serivce_archive_path/bento_archive_path